wordpress/wp-content/plugins/sil-dictionary-webonary/webonary/Webonary_Pathway_Xhtml_Import.php

```diff
@@ -1702,6 +1686,33 @@ public static function redirectToAdminHome()
 		header('Location: ' . $url);
 	}
 
+	/**
+	 * Utility function to convert psuedo-links in entry xml from FLex into actual Webonary site links
+	 * * @param string $entry_xml
+	 * @return string $entry_xml
+	 */	
+	public static function fix_entry_xml_links($entry_xml)
+	{
+		//this replaces a link like this: <a href="#gcec78a67-91e9-4e72-82d3-4be7b316b268">
+		//to this: <a href="/gcec78a67-91e9-4e72-82d3-4be7b316b268">
+		//but it will keep a link like this href="#gcec78a67-91e9-4e72-82d3-4be7b316b268" (important for playing audio)
```

Comment:
This comment needs clarification. Both cases have identical href values.

Reply:
It sure does :-) I tend to ignore stuff like this from legacy code, but it's good you caught it. Fixed it to explain the logic here.